typings/index.d.ts

```diff
@@ -234,6 +232,15 @@ declare module 'discord.js' {
 		public once(event: string, listener: Function): this;
 	}
 
+	export class ClientVoiceManager {
+		constructor(client: Client);
+		public client: Client;
```

Comment:
Maybe make the property readonly and non enumerable via `Object.assign` to be consistent with the rest of the library?
```suggestion
		public client readonly: Client;
```